release/OpenWrt/files/hetzner_ddns

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh /etc/rc.common
```

Comment:
This file should probably be moved to root of the repository, alongside other service files, like this one for [NetBSD](https://github.com/filiparag/hetzner_ddns/blob/0.2.5/hetzner_ddns.netbsd.rc).

That way it can be installed using the combined Makefile by adding a new [target](https://github.com/filiparag/hetzner_ddns/blob/0.2.5/Makefile#L13-L28):

```shell
# OpenWrt service
sudo make openwrt-rc
```

Reply:
Partly fixed with [bea11d1](https://github.com/filiparag/hetzner_ddns/pull/18/commits/bea11d13da88fc315202d75d320456a2b4a1dcd4). Regarding the *hetzner_ddns* `Makefile` please see below.